tests/docker-stacks-foundation/test_packages.py

```diff
@@ -127,9 +118,7 @@ def _check_import_package(
     """Generic function executing a command"""
     LOGGER.debug(f"Trying to import a package with [{command}] ...")
     exec_result = package_helper.running_container.exec_run(command)
-    assert (
-        exec_result.exit_code == 0
-    ), f"Import package failed, output: {exec_result.output}"
+    assert exec_result.exit_code == 0, exec_result.output.decode("utf-8")
```

Comment:
This `.decode("utf-8")` is the most important part of this PR - currently, logs are difficult to read, because they are printed as binary strings.
I've checked after change and logs are much better if some import fails.